src/engine.cpp

```diff
@@ -94,3 +109,9 @@ std::string RandomEngine::getString(){
         }
     return password;
 } 
+
+RandomEngine::~RandomEngine(){
```

Comment:
Even if we want to destruct objects, we should get rid of `new` and `delete` pairs completely by using smart pointers.

Reply:
We would not even need to define a destructor.

Reply:
Ok, so I should use smart pointers instead of ```new``` and ```delete``` pairs. Changes coming soon.

Reply:
> We would not even need to define a destructor.

Yes, sir I have used smart pointers.